pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>org.openhab.tools</groupId>
   <artifactId>static-code-analysis</artifactId>
-  <version>0.0.3</version>
+  <version>0.0.4-SNAPSHOT</version>
```

Comment:
You have your argument for this change: 
>The pom-version is a final one and not a snapshot, openHAB also uses the rule that normally its a snapshot

I think this might be a little bit inconvenient in our case for the people that develop new checks (as the OH2 addons build is using final version in the build). How are they supposed to execute them on the OH2 repo before merge ?